komga-webui/src/locales/en.json

```diff
@@ -166,12 +165,12 @@
   },
   "browse_readlist": {
     "edit_elements": "Edit elements",
-    "edit_readlist": "Edit read list"
+    "edit_readlist": "Edit reading list"
   },
   "browse_series": {
-    "earliest_year_from_release_dates": "This is the earliest year from the release dates from all books in the series",
-    "series_no_summary": "This series has no summary, so we picked one for you!",
-    "summary_from_book": "Summary from book {number}:"
+    "earliest_year_from_release_dates": "Earliest year of any released book in this series.",
+    "series_no_summary": "This series has no summary, so one was randomly selected from one of the series books.",
+    "summary_from_book": "Summary of book {number}:"
```

Comment:
The summary is taken from that book, why change it to "of" ?

Reply:
Because it isn't a summary found within it, of which there could be many for many different things. Yes?

Reply:
that doesn't ring right to me

Reply:
You get sentences like https://www.enotes.com/homework-help/what-is-a-brief-summary-of-book-2-of-the-odyssey-326842

"My summary from book 2 is" vs "My summary of book 2".